src/components/AddForm.js

```diff
@@ -44,6 +44,8 @@ export default function AddForm() {
         />
         <select className={styles.selectInput} name="category" onChange={changeHandler}>
```

Comment:
- [ ] I kindly suggest that you please ensure that the Author state has it's inputs that updates the local state every time it's updated.

Reply:
Thank you very much for your comments!!!